MyAutoBio/coding-test-laravel
# [ADD] Customer Search Functionality

# Description

This PR will add a new functionality to search from this 3 tables

- Customer
- Order
- Item Purchased

## New Features

- [ x ] Docker setup
- [ x ] Feature
- [ x ] PHPCS

### How to test

To setup your environment, you should need to use docker, so run:
```shell
$ docker-compose up -d
$ docker exec -u www-data -it coding_test_laravel_backend bash
www-data@1as:~/html$ cp .env.example .env
www-data@1as:~/html$ php artisan migrate:refresh --seed
www-data@1as:~/html$ npm install
www-data@1as:~/html$ npm run dev
```

Go to the browser to this [link](http://localhost)
